<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1441</width>
    <height>912</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(171, 171, 171);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#000000;&quot;&gt;Список курьеров&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="courier_list">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="courier_info">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(48, 48, 48);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Segoe UI Symbol&quot;;</string>
         </property>
         <property name="text">
          <string>только незанятые курьеры</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Список заказов&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="order_list">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="unsorted_orders">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Segoe UI Symbol&quot;;</string>
           </property>
           <property name="text">
            <string>неподтверждённые заказы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="canceled_orders">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>отклонённые заказы</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sorted_orders">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>подтвержденные заказы</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QTextEdit" name="about_order">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(47, 47, 47);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="confirm_btn">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(67, 67, 67);
</string>
       </property>
       <property name="text">
        <string>Подтвердить заказ</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="exp_lbl">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Нажмите на кнопку, чтобы подтвердить заказ</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="cancel_order">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(67, 67, 67);
</string>
       </property>
       <property name="text">
        <string>Отклонить заказ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Desktop/prefix/resource.qrc"/>
 </resources>
 <connections/>
</ui>
